### PR TITLE
Add method to update axis labels [re #6944]

### PR DESCRIPTION
**Pull request by choochburger**
This adds an `updateAxisLabels` method to `Chart` so that we can update axis labels on the fly without having to re-render the entire Chart.

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -27,5 +27,8 @@
   "node": false,
   "sub": true,
   "undef": true,
-  "unused": "vars"
+  "unused": "vars",
+  "globals": {
+    "jasmine": false
+  }
```
Comment (choochburger) [inline .jshintrc:33]: Fixes a jshint failure that's been hanging out on the feature branch

File: lib/presenter/Chart.js [hunk 2]
```diff
@@ -242,6 +242,13 @@ class Chart extends Events {
         return scales;
     }
 
+    updateAxisLabels(newLabels) {
+       Object.assign(this.axisLabels, newLabels);
+       console.log('labels', this.axisLabels);
```
Comment (choochburger) [inline lib/presenter/Chart.js]: oops, will remove

File: test/presenter/Chart.spec.js [hunk 3]
```diff
@@ -684,9 +697,9 @@ function (
 
 
         describe('HTML smoke tests', function() {
```
Comment (choochburger) [inline test/presenter/Chart.spec.js:699]: These updates are just to include the new `axis_label--x` and `axis_label--y` classes

File: .jshintrc (hunk 1 above)
Comment (nickdoesdesign) [reply .jshintrc:33]: Nice.

**Pull request merged**